packages/at_chops/lib/src/algorithm/ecc_signing_algo.dart

```diff
@@ -29,7 +29,8 @@ class EccSigningAlgo implements AtSigningAlgorithm {
   @override
   bool verify(Uint8List signedData, Uint8List signature, {String? publicKey}) {
     if (publicKey == null && _privateKey == null) {
-      throw AtSigningVerificationException('public key not set not ecc verification');
+      throw AtSigningVerificationException(
+          'public key not set not ecc verification');
```

Comment:
1. I don't really understand the exception message here. Should it be, "Cannot verify signature - publicKey not provided or privateKey not available"
2. Should it be an `||` condition rather than an `&&`?

Reply:
Got it - makes sense. Can you update the exception message as I suggested please?

Reply:
done